### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by rustamwin**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,37 +1,45 @@
...
-        "php": "^7.2"
+        "php": "^7.4|^8.0",
+        "yiisoft/auth": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.2",
-        "phan/phan": "^2.2 || ^3.0"
+        "infection/infection": "^0.17.0",
+        "phpunit/phpunit": "^9.3",
+        "nyholm/psr7": "^1.0",
```
Comment (roxblnfk) [inline composer.json:27]: Is this requirement relevant now?

File: composer.json [hunk 2]
```diff
@@ -1,37 +1,45 @@
...
+        "vimeo/psalm": "^3.15"
     },
     "autoload": {
         "psr-4": {
-            "Yiisoft\\_____\\": "src"
+            "Yiisoft\\AuthJwt\\": "src"
```
Comment (roxblnfk) [inline composer.json:32]:
```suggestion
            "Yiisoft\\Auth\\Jwt\\": "src"
```

File: composer.json [hunk 3]
```diff
@@ -1,37 +1,45 @@
...
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "Yiisoft\\_____\\Tests\\": "tests"
+            "Yiisoft\\AuthJwt\\Tests\\": "tests"
```
Comment (roxblnfk) [inline composer.json:37]:
```suggestion
            "Yiisoft\\Auth\\Jwt\\Tests\\": "tests"
```

File: composer.json (hunk 1 above)
Comment (rustamwin) [reply composer.json:27]: Did you mean to replace it with httpsoft?

Comment (roxblnfk) [reply composer.json:27]: Not. Why add a dependency if it's not used in this code?

Comment (rustamwin) [reply composer.json:27]: This will be used for tests

Comment (rustamwin) [reply composer.json]: OK. Let's remove it for now.

**Review by romkatsu — APPROVED** [review]

**Review by roxblnfk — APPROVED** [review]

**Pull request merged**